erpnext/translations/fr.csv

```diff
@@ -2811,7 +2811,7 @@ Stock Value,Valeur du Stock,
 Stock balance in Batch {0} will become negative {1} for Item {2} at Warehouse {3},Solde du stock dans le Lot {0} deviendra négatif {1} pour l'Article {2} à l'Entrepôt {3},
 Stock cannot be updated against Delivery Note {0},Stock ne peut pas être mis à jour pour le Bon de Livraison {0},
 Stock cannot be updated against Purchase Receipt {0},Stock ne peut pas être mis à jour pour le Reçu d'Achat {0},
-Stock cannot exist for Item {0} since has variants,Stock ne peut pas exister pour l'Article {0} puisqu'il a des variantes,
+Stock cannot exist for Item {0} since has variants,Stock ne peut pas exister pour l&#39;Article {0} puisqu'il a des variantes,
```

Comment:
Is `&#39;` required? I tried making this change locally and:
<img width="691" alt="Screenshot 2022-06-09 at 7 33 42 PM" src="https://user-images.githubusercontent.com/25857446/172866280-b56a3c7d-3db8-4606-8b5e-a3a62b934cc2.png">

l'Article works just fine right ?

Reply:
Some translation can work with ' others need &#39; 
Not related to french language but frappe js framework that newly deal strangly with quote in translation

For example (actual translation)
Stock Ledger,Livre d'Inventaire,
cause the label of menu item have simple quote a there is a js errors (as you can see there is missing button because js crash)
![image](https://user-images.githubusercontent.com/1050053/172881573-68dd9cda-2ddc-4446-844d-9b26706128fd.png)

That why I suggest 
Stock Ledger,Livre d&#39;Inventaire,
![image](https://user-images.githubusercontent.com/1050053/172883250-429dde51-ff52-4848-ad8c-5332fafcc1ae.png)


I hesitate to review all french translation to change simple quote by &#39; But I prefers to go one by one because this always works

Where your right is that I let a simple quote into the same translation, I've just update it

Reply:
You'r right it seems that my quote problem only affect doctype name (that have to be displayed into View or Action card button), so I let the normal simple quote on this reviewed line sentence, but the problem for field name or doctype name is still there, I jut have to check where the translation is used

Reply:
@marination I've send a fix for the simple quote button name problem into frappe, So I re-change `&#39;` to simple quote. Better that way

Reply:
@FHenry Awesome !! 😄